pywbemtools/pywbemcli/_cmd_server.py

```diff
@@ -250,12 +251,13 @@ def cmd_server_info(context):
     """
     Display general overview of info from current WBEM server
     """
+    wbem_server = context.pywbem_server.wbem_server
     output_format = validate_output_format(context.output_format, 'TABLE')
 
     try:
-        # execute the namespaces to force contact with server before
+        # Execute the namespaces to force contact with server before
         # turning off the spinner.
-        server = context.wbem_server
+        server = wbem_server
```

Comment:
For consistency, I suggest to eliminate the additional `server` variable and go with `wbem_server` directly.

Reply:
Done